bin/start_backend.sh

```diff
@@ -164,6 +167,12 @@ if [ ! -z ${UDF_RUNTIME_DIR} ]; then
     rm -f ${UDF_RUNTIME_DIR}/*
 fi
 
+chmod 755 ${STARROCKS_HOME}/lib/starrocks_be
+if [ ${RUN_META_TOOL} -eq 1 ] ; then
```

Comment:
Should be read only, otherwise there is a risk here

Reply:
done